scripts/nginx/cgi-bin.conf

```diff
@@ -0,0 +1,7 @@
+location /cgi-bin/ {
+  gzip off;
+  root  /var/www/html;
+  fastcgi_pass  unix:/var/run/fcgiwrap.socket;
+  include /etc/nginx/fastcgi_params;
+  fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
```

Comment:
Should this be `__SCRIPT_FILENAME__`?

Reply:
no, SCRIPT_FILENAME is a nginx identifier.